airlock/file_browser_api.py

```diff
@@ -17,58 +18,72 @@ class PathItem:
            something which is not tied to concrete filesystem paths.
     """
 
+    class PathNotFound(Exception):
+        pass
+
     container: AirlockContainer
-    relpath: UrlPath = ROOT_PATH
+    relpath: UrlPath
+
+    type: str = None
```

Comment:
I'm never 100% sure when an enum is a good thing, but would it make sense here? PathItem type should only ever be dir or file, shouldn't it?

Reply:
Oh, no, I see it can also be workspace/request - maybe an enum is useful then, so it's clear what the possible values are

Reply:
Yes, I think you're right, I will add an enum :+1: